source/jupyter-book-tutorials.rst

```diff
@@ -0,0 +1,8 @@
+Jupyter Book Tutorials
+============================
+
+These tutorials are powered by `Jupyter Book <https://jupyterbook.org/intro.html>`_
+with support from the `Sloan Foundation <https://sloan.org/grant-detail/9231>`_.
+
```

Comment:
Let's swap the order of the two documents, since the CMIT is more complete than q2book.